src/DynamoCore/Graph/Nodes/DummyNode.cs

```diff
@@ -370,6 +466,16 @@ internal string GetDescription()
         /// </summary>
         public string LegacyAssembly { get; private set; }
 
+        /// <summary>
+        /// Type name of the node.
+        /// </summary>
+        public string TypeName { get; private set; }
+
+        /// <summary>
+        /// Returns the node's function name
+        /// </summary>
+        public string FunctionName { get; private set; }
```

Comment:
any reason not to overload the LegacyNodeName - maybe note that this is property is only valid for ZeroTouch dummy nodes?